### PR TITLE
Remove connect-memjs and set default to use data source from database (instead of cache)

### PR DESCRIPTION
**Pull request by garrettwong**
Remove connect-memjs (#7 )
and set default to use data source from database (instead of cache) (#13 )

### CODE REVIEW

File: infrastructure/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+./build-images.sh
+
+# create a file named "source.env"
+cat > source.env << EOF
+export CLOUDSQL_HOSTNAME="1.2.34.56"
+export CLOUDSQL_USERNAME="user"
+export CLOUDSQL_PASSWORD="password"
+export CLOUDSQL_SCHEMA="forseti_security"
+export FORSETI_SERVER_VM_CHANNEL="1.2.3.4:50051"
+export FORSETI_DATA_MODEL_HANDLE="abcdaa3a5bc9fd9acfaf50fdd3620534"
```
Comment (joecheuk) [inline infrastructure/README.md:25]: Seems like we need to manually export the model handle, is this a manual step user needs to do everytime they want to update w/ the latest inventory?

**Review by joecheuk — APPROVED** [review]

File: infrastructure/README.md (hunk 1 above)
Comment (garrettwong) [reply infrastructure/README.md:25]: At this time, yes, otherwise, it will be sourcing explain data from an older version.

**Comment by garrettwong** [comment]
Resolves #7 
Resolves #13

**Comment by garrettwong** [comment]
Resolves #10

**Pull request merged**